addons/website/views/website_templates.xml

```diff
@@ -2601,6 +2601,96 @@ Sitemap: <t t-esc="url_root"/>sitemap.xml
     </xpath>
 </template>
 
+<template id="step_wizard" name="Step Checkout">
```

Comment:
It's only used in ecommerce, no?

Reply:
Hello @rdeodoo :wave: Indeed, it is currently only used in e-commerce. @mano-odoo wanted to put that here forecasting a design change in appointment and event. I'll let him defend his idea.